cypress/integration/shared.js

```diff
@@ -191,3 +191,11 @@ export const waitForClusterInstallation = () => {
   // wait up to 1 hour for the progress description to say "Installed"
   cy.contains('div.pf-c-progress__description', 'Installed', { timeout: CLUSTER_CREATION_TIMEOUT });
 };
+
+export const saveClusterDetails = () => {
```

Comment:
I prefer to pass the `cy` as a parameter here, it will provide flexibility in the future.